docs/ABI/TypeMetadata.rst

```diff
@@ -222,13 +222,17 @@ classes.
   record is stored at **offset 0**, in place of an integer kind discriminator.
 - The **super pointer** pointing to the metadata record for the superclass is
   stored at **offset 1**. If the class is a root class, it is null.
-- Two words are reserved for use by the Objective-C runtime at **offset 2**
-  and **offset 3**.
-- The **rodata pointer** is stored at **offset 4**; it points to an Objective-C
-  compatible rodata record for the class. This pointer value includes a tag.
+- If there is Objective-C interoperability, two words are reserved for use by 
```

Comment:
I'd suggest something like, "On platforms which support Objective-C interoperability...." Just to make it clear that this is part of the platform, and not something that can be chosen on a per-program or per-type basis.